src/opm/input/eclipse/EclipseState/Grid/NNC.cpp

```diff
@@ -171,14 +177,81 @@ bool is_neighbor(const EclipseGrid& grid, std::size_t g1, std::size_t g2) {
         }
     }
 
+    void NNC::load_editr(const EclipseGrid& grid, const Deck& deck) {
+        // Will contain EDITNNCR data in reverse order to be able
+        // use unique to only keep the last one from the data file.
+        std::deque<NNCdata> nnc_editr;
+
+        const auto& keyword_list = deck.getKeywordList<ParserKeywords::EDITNNCR>();
+
+        if (keyword_list.empty()) {
+            return;
+        }
+
+        for (const auto& keyword_ptr : deck.getKeywordList<ParserKeywords::EDITNNCR>()) {
+            const auto& records = *keyword_ptr;
+            if (records.empty()) {
+                continue;
+            }
+            for (const auto& record : records) {
+                auto index_pair = make_index_pair(grid, record);
+                if (!index_pair)
+                    continue;
+
+                auto [g1, g2] = index_pair.value();
+                if (is_neighbor(grid, g1, g2))
+                    continue;
+
+                double trans = record.getItem(6).getSIDouble(0);
+                nnc_editr.emplace_front(g1, g2, trans);
+            }
+
+            if (!this->m_editr_location)
+                this->m_editr_location = keyword_ptr->location();
+        }
+
+        if (nnc_editr.empty()) {
+            return;
+        }
+
+        // Make the entries unique (i.e. only one entry for each index pair)
+        // the entry surviving should be last one (i.e. later enrtries overwrite previous entries)
+        // Will keep the insertion order of entries for same cell pair.
+        std::sort(nnc_editr.begin(), nnc_editr.end());
+
+        // remove duplicates (based on only cell1 and cell2 members
+        auto equality = [](const NNCdata& d1, const NNCdata& d2){
+            return d1.cell1 == d2.cell1 && d1.cell2 == d2.cell2;
+        };
+        auto new_end = std::unique(nnc_editr.begin(), nnc_editr.end(), equality);
+        nnc_editr.resize(new_end - nnc_editr.begin());
```

Comment:
I'd probably use `deque::remove()` here:
```C++
nnc_editr.remove(new_end, nnc_editr.end());
```

Reply:
That function doesn't seem to exist. Do you mean `deque::erase`? 
I am reading something about linear complexity for `deque::erase` (whereas `deque::resize` should be constant).
Please clarify what you mean and why that is a good idea. I am probably missing something.

For now I am leaving this as it is.

Reply:
> [`deck::remove()`] doesn't seem to exist. Do you mean `deque::erase()`?

Yes, I did mean `deck::erase()`.

> I am reading something about linear complexity for `deque::erase` (whereas `deque::resize` should be constant).

`Deck::resize()` is linear complexity too.  It needs to call constructors/destructors for all new/removed elements.

> Please clarify what you mean and why that is a good idea. I am probably missing something.

It's more explicit.  Moreover, `erase()` will only ever remove elements, whereas `resize()` has to account for the possibility of new elements being added.